### ISSUE TITLE
Bloom bug after 95%

### ISSUE THREAD

**Issue by kpot87** [albertobsd/keyhunt#122]
Hi , when the bloom is creating after 95% all -t drops to just 1 core and last 5% take too long time to complete. This is a bug? Or it’s normal? Thanks

**Comment by sssergy** [comment]
> Привет, когда цветение создается после 95% всего -t падает до 1 ядра, а последние 5% занимают слишком много времени для завершения. Это ошибка? Или это нормально? Спасибо

I have already asked this question, but I do not know English. Perhaps not so asked, but there is no answer yet.

**Comment by kpot87** [comment]
@sssergy привет, псу тебя тоже такая ерунда

**Comment by albertobsd** [comment]
Yes it's some weird but, i already manage to solve it i will be solved in the next update i hope upload it today or tomorrow.

Regards!

**Comment by albertobsd** [comment]
It was solved with the new updated?
